tests/Unit/TicketTest.php

```
@@ -0,0 +1,149 @@
+	// -------------------------------------------------------------------------
+
+	/**
+	 * Sanitize_text_field preserves spaces in original filenames, ensuring the
+	 * stored origname matches what the user uploaded.
+	 */
+	public function test_sanitize_text_field_preserves_spaces(): void {
+		$original = 'My Support File (v2).pdf';
+		$result   = sanitize_text_field( $original );
+		$this->assertStringContainsString( ' ', $result, 'sanitize_text_field should preserve spaces' );
+		$this->assertSame( $original, $result );
+	}
```

PHPMD (2.15.0)

[error] The method test_sanitize_text_field_preserves_spaces is not named in camelCase. (undefined)

(CamelCaseMethodName)